widget/playground/src/containers/StyleLayout/StyleLayout.CustomColors.tsx

```diff
@@ -0,0 +1,220 @@
+import type { CustomColorsTypes } from './StyleLayout.types';
+import type { Mode } from '../../store/config';
+import type { WidgetColorsKeys } from '@rango-dev/widget-embedded';
+
+import {
+  ChevronDownIcon,
+  ChevronUpIcon,
+  Collapsible,
+  CustomColorsIcon,
+  Divider,
+  Typography,
+} from '@rango-dev/ui';
+import React, { useEffect, useState } from 'react';
+
+import { ColorItemPicker } from '../../components/ColorItemPicker';
+import { WIDGET_COLORS } from '../../constants';
+import { useConfigStore } from '../../store/config';
+
+import {
+  Color,
+  ColorsContent,
+  CustomColorCollapsible,
+  CustomColors,
+  FieldTitle,
+  Row,
+} from './StyleLayout.styles';
+
+const Colors = (props: { mainColor?: string; secondColor?: string }) => {
+  return (
+    <ColorsContent>
+      <Color style={{ backgroundColor: props.mainColor }} />
+      {props.secondColor !== undefined && (
+        <Color
```

Comment:
can we remove these inline styles?

Reply:
I think we have a component called Color in the ui package, can we have another name for this component?
Also I saw these hard codes related to previous PR. we can read them from theme:
![image](https://github.com/rango-exchange/rango-client/assets/123297941/40163054-d9fd-4d3e-9254-90352288ab08)

Reply:
fixed